coalib/output/printers/LogPrinter.py

```diff
@@ -117,7 +120,10 @@ def __init__(self,
         :param timestamp_format: The format string for the
                                  datetime.today().strftime(format) method.
         """
-        self.logger_name = 'coala.raw'
+        if isinstance(printer, ColorPrinter):
+            self.logger_name = 'coala.colored'
+        else:
+            self.logger_name = 'coala.raw'
```

Comment:
this is a bad design pattern, ideally the ColorPrinter does what it needs on its own and this LogPrinter shouldn't have anything to do with it